### PR TITLE
Fix webhook payload save on Azure Storage

### PR DESCRIPTION
**Pull request by cmiacz**
* Fix saving webhook to Azure Storage by using bytes rather than string

* Test reading event payload

* Add changelog for azure storage webhook fix

---------

I want to merge this change because...

# Impact

- [ ] New migrations
- [ ] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
- [ ] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [ ] The changes are covered by test cases
- [ ] All new fields/inputs/mutations have proper labels added (`ADDED_IN_X`, `PREVIEW_FEATURE`, etc.)
- [ ] All migrations have proper dependencies
- [ ] All indexes are added concurrently in migrations
- [ ] All RunSql and RunPython migrations have revert option defined

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2685,3 +2685,8 @@ Just so you know, changes mentioned in this section are in a preview state and c
 
 - Add createdb note to getting started for Windows - #3106 by @ajostergaard
 - Update docs on pipenv - #3045 by @jxltom
+- Added support for numeric and lower-case boolean environment variables - #16313 by @NyanKiyoshi
+- Fixed a potential crash when Checkout metadata is accessed with high concurrency - #16411 by @patrys
+- Add slugs to product/category/collection/page translations. Allow to query by translated slug - #16449 by @delemeator
+- Fixed a crash when the Decimal scalar is passed a non-normal value - #16520 by @patrys
+- Fixed a bug when saving webhook payload to Azure Storage - #16585 by @delemeator
```
Comment (maarcingebala) [inline CHANGELOG.md:2692]: This seems like not part of this PR.

**Review by wcislo-saleor — APPROVED** [review]

**Review by przlada — APPROVED** [review]

**Review by korycins — APPROVED** [review]
